src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3Driver.cs

```diff
@@ -141,6 +141,7 @@ RaspberryBoardInfo.Model.RaspberryPi3BPlus or
             RaspberryBoardInfo.Model.RaspberryPiZeroW or
             RaspberryBoardInfo.Model.RaspberryPiZero2W or
             RaspberryBoardInfo.Model.RaspberryPi4 or
+            RaspberryBoardInfo.Model.RaspberryPi5 or

```

Comment:
Thanks... Done... I've also added the Pi5 Board reference to the GpioController class along with a switch choosing the LibGpiodDriver when on windows.

However, I guess this will only be used if running on Windows 10 or the now (pretty much) defunct Windows 10 IoT?

Do let me know if you think this should be removed though?